libs/hdf-converters/src/mappings/ChecklistMappingData.ts

```diff
@@ -0,0 +1,241 @@
+import _ from 'lodash';
+
+export type ChecklistFile = {
+  /**
+   * Unique identifier for this file. Used to encode which file is currently selected, etc.
+   *
+   * Note that in general one can assume that if a file A is loaded AFTER a file B, then
+   * A.unique_id > B.unique_id.
+   * Using this property, one might order files by order in which they were added.
+   */
+  uniqueId: string;
```

Comment:
some of the types here seem to be added in to make it ingestible within heimdall.  now that we're changing the implementation to use hdf internally for the checklist viewer and stuff, let's get rid of the non-checklist types

Reply:
Removed